content/influxdb/v2.0/reference/cli/influx/v1/auth/list.md

```diff
@@ -37,3 +37,32 @@ influx v1 auth list [flags]
 | `-u` | `--user`          | InfluxDB user                                                            | string     |                         |
 |      | `--user-id`       | InfluxDB user ID                                                         | string     |                         |
 |      | `--username`      | Authorization username                                                   | string     | `$INFLUX_USERNAME`      |
+
+## Examples
+
+{{< cli/influx-creds-note >}}
+
+- [List all v1 authorizations](#list-all-v1-authorizations)
+- [List v1 authorizations associated with a user](#list-v1-authorizations-associated-with-a-user)
+- [List v1 authorizations associated with a user ID](#list-v1-authorizations-associated-with-a-user-id)
+- [List a specific v1 authorization by ID](#list-a-specific-v1-authorization-by-id)
```

Comment:
Do these list authentication tokens?

Reply:
No, v1 authorizations are different than authentication tokens... yeah... the water is muddy.

Reply:
added to list -perhaps covered by our conceptual discussion of auth :-)